### PR TITLE
Added additional information about ANALYZE TABLE in TiDB

### LINKED ISSUE #833: Add section on mysql compatibility page: ANALYZE TABLE command
Analyze table in TiDB takes *significantly* longer in TiDB than MySQL (not a bad thing in itself - it builds different/better statistics), whereas MySQL(InnoDB) just samples random pages.

It should be documented, since some naive scripts may need to be changed (i.e. expecting ANALYZE to take less than a second, when it takes 30 minutes..)

### PR DESCRIPTION
**Pull request by kolbe**
Added additional information about ANALYZE TABLE in TiDB

Fixes #833.

### CODE REVIEW

File: sql/statistics.md [hunk 1]
```diff
@@ -15,7 +15,7 @@ Based on the statistics, the TiDB optimizer chooses the most efficient query exe
 
 ### Manual collection
 
-You can run the `ANALYZE` statement to collect statistics.
+You can run the `ANALYZE` statement to collect statistics. Note that `ANALYZE TABLE` in TiDB takes considerably longer than in MySQL/InnoDB. In InnoDB, only a small number of pages are sampled, while in TiDB a comprehensive set of statistics is completely rebuilt. Many scripts written for MySQL execute `ANALYZE TABLE` expecting that it will be a short-lived operation; those scripts should be modified not to make that same assumption about TiDB.
```
Comment (morgo) [inline sql/statistics.md]:
```suggestion
You can run the `ANALYZE` statement to collect statistics. Note that `ANALYZE TABLE` in TiDB takes considerably longer than in MySQL/InnoDB. In InnoDB, only a small number of pages are sampled, while in TiDB a comprehensive set of statistics is completely rebuilt. Scripts that were written for MySQL may naively expect `ANALYZE TABLE` will be a short-lived operation.
```

**Comment by morgo** [comment]
I think this is significant enough, we should add it on the mysql-compatibility page as well under differences.

**Comment by morgo** [comment]
LGTM

**Comment by morgo** [comment]
I have moved `EXPLAIN` under database administration as well.  It doesn't need its own heading.

**Comment by ceohockey60** [comment]
LGTM

File: sql/mysql-compatibility.md [hunk 2]
```diff
@@ -99,6 +99,14 @@ TiDB implements the asynchronous schema changes algorithm in F1. The Data Manipu
 + Rename Table
 + Create Table Like
 
+### Database Administration
+
+Many administrative statements in TiDB work as they do in MySQL, but there are some important differences:
+
++ [`ANALYZE TABLE`](statistics.md#manual-collection) works differently in TiDB than in MySQL, in that it's a relatively lightweight and short-lived operation in MySQL/InnoDB, while in TiDB it completely rebuilds the statistics for a table and can take much longer to complete.
```
Comment (lilin90) [inline sql/mysql-compatibility.md]: This is a broken link at the PingCAP website. For link path, please start from the root directory. Two choices:

- From the root directory to the current file: `/sql/statistics.md`
- From the current file to the root directory: `../sql/statistics.md`

**Pull request merged**